---
title: Bloku catch nebylo nikdy dosaženo; <exception>zpracované výše ve stejném příkazu try
ms.date: 07/20/2015
f1_keywords:
- bc42031
- vbc42031
helpviewer_keywords:
- BC42031
ms.assetid: 7d15597c-5988-42ea-a853-63cbf78faaf3
ms.openlocfilehash: 987a509b4b099a0a1f728f55dfb2b35a8770af3f
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: cs-CZ
ms.lasthandoff: 06/04/2020
ms.locfileid: "84395477"
---
# <a name="catch-block-never-reached-exception-handled-above-in-the-same-try-statement"></a><span data-ttu-id="e3040-102">Bloku catch nebylo nikdy dosaženo; \<exception>zpracované výše ve stejném příkazu try</span><span class="sxs-lookup"><span data-stu-id="e3040-102">'Catch' block never reached; \<exception> handled above in the same 'Try' statement</span></span>
<span data-ttu-id="e3040-103">`Catch`Blok v kódu není dostupný, protože se zpracovává v předchozím `Try` bloku.</span><span class="sxs-lookup"><span data-stu-id="e3040-103">A `Catch` block in the code cannot be reached because it is handled in a preceding `Try` block.</span></span>  
  
<span data-ttu-id="e3040-104">Ve výchozím nastavení je tato zpráva upozornění.</span><span class="sxs-lookup"><span data-stu-id="e3040-104">By default, this message is a warning.</span></span> <span data-ttu-id="e3040-105">Další informace o skrývání upozornění nebo zpracování upozornění jako chyb najdete v tématu [Konfigurace upozornění v Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).</span><span class="sxs-lookup"><span data-stu-id="e3040-105">For more information about hiding warnings or treating warnings as errors, see [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).</span></span>
  
 <span data-ttu-id="e3040-106">**ID chyby:** BC42031</span><span class="sxs-lookup"><span data-stu-id="e3040-106">**Error ID:** BC42031</span></span>  
  
## <a name="to-correct-this-error"></a><span data-ttu-id="e3040-107">Oprava této chyby</span><span class="sxs-lookup"><span data-stu-id="e3040-107">To correct this error</span></span>  
  
- <span data-ttu-id="e3040-108">Odeberte redundantní příkaz.</span><span class="sxs-lookup"><span data-stu-id="e3040-108">Remove the redundant statement.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="e3040-109">Viz také</span><span class="sxs-lookup"><span data-stu-id="e3040-109">See also</span></span>

- [<span data-ttu-id="e3040-110">Jak zachytit výjimky pomocí bloku try/catch</span><span class="sxs-lookup"><span data-stu-id="e3040-110">How to use the try/catch block to catch exceptions</span></span>](../../standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.md)
- [<span data-ttu-id="e3040-111">Používání obslužných rutin uživatelsky filtrovaných výjimek</span><span class="sxs-lookup"><span data-stu-id="e3040-111">Using User-Filtered Exception Handlers</span></span>](../../standard/exceptions/using-user-filtered-exception-handlers.md)
- [<span data-ttu-id="e3040-112">Try...Catch....Finally – příkaz</span><span class="sxs-lookup"><span data-stu-id="e3040-112">Try...Catch...Finally Statement</span></span>](../language-reference/statements/try-catch-finally-statement.md)
